core/templates/dev/head/pages/topic-viewer-page/skills-list/concept-card-modal.template.html

```diff
@@ -0,0 +1,11 @@
+<div class="modal-header">
```

Comment:
Can we reuse the concept card modal that the question player uses? I think the only difference is the footer buttons right? We can pass the footer button data to the modal

Reply:
What did you mean by "pass the footer button data to the modal"?

Reply:
Done.